src/actions/guides/http_and_routing/request_and_response.cr

```diff
@@ -7,6 +7,46 @@ class Guides::HttpAndRouting::RequestAndResponse < GuideAction
 
   def markdown
     <<-MD
+    ## Handling Requests
+
+    When a request comes in to an Action, Lucky gives you access to the request object through the `request` method.
+    > The `request` object is an instance of [HTTP::Request](https://crystal-lang.org/api/HTTP/Request.html).
+
+    Lucky gives you access to a few helpful methods based on the request Content-Type.
```

Comment:
Might be worth adding an example of using this in an action (like accessing headers)

Also might be worth mentioning you have access to the response object too but usually isn’t be used since we have helper methods. But can for fancier stuff

Reply:
There's an example of these right below where they are [defined](https://github.com/paulcsmith/website-v2/pull/32/files#diff-6c5ae151a82b9614255254411efcd54aR25). 

I'll add a mention for the response object too. That could be handy.

Reply:
oh.. I wonder if you meant to comment on the next section below. I just saw where I missed an example.

Reply:
Oops. I meant to make this comment on the paragraph above that says you have access to the request. So you could show an example of accessing the request headers or something